cumulus_etl/loaders/fhir/ndjson_loader.py

```diff
@@ -112,3 +114,28 @@ async def load_from_bulk_export(
             group_name=bulk_exporter.group_name,
             export_datetime=bulk_exporter.export_datetime,
         )
+
+    def read_deleted_ids(self, root: store.Root) -> dict[str, set[str]]:
+        """
+        Reads any deleted IDs that a bulk export gave us.
+
+        See https://hl7.org/fhir/uv/bulkdata/export.html for details.
+        """
+        deleted_ids = {}
+
+        subdir = store.Root(root.joinpath("deleted"))
+        bundles = common.read_resource_ndjson(subdir, "Bundle")
+        for bundle in bundles:
+            if bundle.get("type") != "transaction":
+                continue
+            for entry in bundle.get("entry", []):
+                request = entry.get("request", {})
+                if request.get("method") != "DELETE":
+                    continue
+                url = request.get("url")  # should be relative URL like "Patient/123"
+                if not url or url.count("/") != 1:
```

Comment:
This was sanity checking. The spec isn't extremely specific on the content of the URL, but gives an example of the 99% case of the relative URL (`Patient/123`).

But like... the content *could* be an absolute URL or any URI format like a `urn:` situation. But if the EHR is being kind to us at all, it would be a normal relative URL. An absolute URL we could also support and just take the last two bits of the path, after we parsed it out? But I felt like that was unlikely enough that we could come to that if we came to that.